### ISSUE TITLE
[Bug]: The PluginView be trigger destroy on create Tiptap instance.

### ISSUE THREAD

**Issue by yangfuhai** [ueberdosis/tiptap#5681]
### Affected Packages

core

### Version(s)

2.7.4

### Bug Description

```ts
export const MyExtension = Extension.create<ClickMenuOptions>({
  name: "MyExtension",
  addProseMirrorPlugins() {
    const destroy = () => {
      console.log("destroy!!!!")
    }
    return [
      new Plugin({
        key: new PluginKey(`MyExtension`),
        view: () => ({ destroy }),
      }),
    ];
  },
});
```

the method `destroy` will be invoked twice in Tiptap create.

>ps: This problem does not exist in tiptap v2.6x.

### Browser Used

Chrome

### Expected Behavior

the destory method not be trigger in tiptap create.

### Dependency Updates

- [X] Yes, I've updated all my dependencies.

**Comment by nperez0111** [comment]
Thanks for bringing this up. Was the new onDrop & onPaste events that were always registering, I have a fix in this https://github.com/ueberdosis/tiptap/pull/5684

**Comment by nperez0111** [comment]
This has been released with tiptap v2.8.0
